ligo/stablecoin/spec.ligo

```diff
@@ -0,0 +1,227 @@
+// SPDX-FileCopyrightText: 2020 tqtezos
+// SPDX-License-Identifier: MIT
+
+/*
+ * Stablecoin smart-contract
+ */
+
+/* ------------------------------------------------------------- */
+
+/*
+ * Parameter
+ */
+
+type tokenId is nat
+
+
+
+type transferDescriptor is record
+  id          : tokenId
+; fromAddress : address
+; toAddress   : address
+; amount      : nat
+end
+
+type transferParams is record
+  batch    : list (transferDescriptor)
+; operator : address
+end
+
+
+
+type balanceOfRequest is record
+  id      : tokenId
+; owner   : address
+end
+
+type balanceOfResponse is record
+  request : balanceOfRequest
+; balance : nat
+end
+
+type balanceOfParams is record
+  requests : list (balanceOfRequest)
+; callback : contract (list (balanceOfResponse))
+end
+
+
+
+type getTotalSupplyResponse is record
+  id          : tokenId
+; totalSupply : nat
+end
+
+type getTotalSupplyParams is record
+  ids      : list (tokenId) // list of requests
+; callback : contract (list (getTotalSupplyResponse))
+end
+
+
+
+type getTokenMetadataResponse is record
+  id       : tokenId
+; symbol   : string
+; name     : string
+; decimals : nat
+; extras   : map (string, string)
+end
+
+type getTokenMetadataParams is record
+  id       : tokenId
+; callback : contract (list (getTokenMetadataResponse))
+end
+
+
+
+type operatorTokens is
+| AllTokens
+| SomeTokens of set (tokenId)
+
+type operatorParam is record
+  owner    : address
+; operator : address
+; tokens   : operatorTokens
+end
```

Comment:
Here, we have the `owner` field as per the FA2 doc. But do you know why this is required? Can this field be anything other than the actual sender of the transaction? What is the behavior in that case?

Reply:
I don't think so too and I think that I better remove these appearances